### PR TITLE
fix aligment to the left

### PR DESCRIPTION
**Pull request by cedvdb**
Fix https://github.com/alex-melnyk/flutter_advanced_drawer/issues/54

Could you publish this when merged ?

### CODE REVIEW

File: lib/src/widget.dart [hunk 1]
```diff
@@ -138,6 +138,7 @@ class _AdvancedDrawerState extends State<AdvancedDrawer>
                 textDirection:
                     widget.rtlOpening ? TextDirection.rtl : TextDirection.ltr,
                 child: ScaleTransition(
+                  alignment: Alignment.centerLeft,
```
Comment (alex-melnyk) [inline lib/src/widget.dart:141]: It looks like it should be based on a widget.rtlOpening` value, isn't?

**Review by alex-melnyk — CHANGES_REQUESTED** [review]
Please pay attention to the comments

File: lib/src/widget.dart (hunk 1 above)
Comment (cedvdb) [reply lib/src/widget.dart:141]: yes

**Comment by cedvdb** [comment]
@alex-melnyk ping for notification: comments are resolved.

**Pull request merged**